python/unityagents/unity_gym_env.py

```diff
@@ -0,0 +1,173 @@
+import gym
+import numpy as np
+from unityagents import UnityEnvironment
+from gym import error, spaces, logger
+
+
+class UnityGymWrapperException(error.Error):
+    """
+    Any error related to the gym wrapper of ml-agents.
+    """
+    pass
+
+
+def multi_agent_check(info):
+    if len(info.agents) != 1:
+        raise UnityGymWrapperException(
+            "There can only be one agent in a UnityEnvironment "
+            "if it is wrapped in a gym.")
+
+
+class UnityEnv(gym.Env):
```

Comment:
Suggest calling this class something more specific, out of context, it is very similar to `UnityEnvironment`